### PR TITLE
Issue: "#3933":Modified the NoLineWrapCheck method to take input from fi…

### PR DESCRIPTION
**Pull request by divsidhu-26**
…les from a subfolder in the directory and also added the input files to that subfolder

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoLineWrap/InputnolinewrapGood.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.NoLineWrap;
+
+/**
+ *
+ * @author divsidhu-26
+ */
+import com.puppycrawl.tools.checkstyle.TreeWalker;
+
+import javax.accessibility.AccessibleAttributeSequence;
+import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoLineWrap/InputnolinewrapGood.java]: @divsidhu-26 Inputs don't go in the `main` folder. They go in the `test` folder under `resources`.

**Comment by divsidhu-26** [comment]
Can you help me with why are the checks failing?

**Comment by MEZk** [comment]
@divsidhu-26 
Please, squash your commit in one.

**Comment by rnveach** [comment]
@divsidhu-26 See my comment at https://github.com/checkstyle/checkstyle/pull/3941#discussion_r104437120
> @divsidhu-26 Inputs don't go in the `main` folder. They go in the `test` folder under resources.

**Comment by divsidhu-26** [comment]
Thank you @MEZk @rnveach ............
I will do it again and make a pull request.......

**Pull request closed without merging**